### ISSUE TITLE
Shopware 6.5 with essential Image not possible

### ISSUE THREAD

**Issue by grosssmutttr** [dockware/dockware#176]
In Shopware 6.5 it is also no longer possible to download the zip file from here:
https://www.shopware.com/en/changelog/

See this:
https://www.shopware.com/en/news/the-improved-shopware-installer/

So it is no longer possible to setup the essential image as described in the docs:
https://docs.dockware.io/tips-and-tricks/create-a-new-project

Additionally if defining a node_version=18 it shows the following in the docker container:
2023-05-26 18:30:32 shopware6-5 | ! WARNING: Version '18' does not exist.
2023-05-26 18:30:32 shopware6-5 | default -> 18 (-> N/A)

`version: "3"

services:
shopware6-5:
image: dockware/essentials:latest
container_name: shopware6-5
ports:
- "80:80"
- "3306:3306"
- "22:22"
- "8888:8888"
- "9999:9999"
- "9998:9998"
networks:
- web
environment:
- XDEBUG_ENABLED=1
- PHP_VERSION=8.1
- NODE_VERSION=18 https://github.com/dockware/dockware/issues/12|14|16|18

networks:
web:
external: false`

PHP_VERSION=8.2 also does not work as it does not find the corresponding php-fpm version.

**Comment by boxblinkracer** [comment]
Hi there

thank you for this
so regarding the things

php 8.2 is coming soon, martin just did a soft launch of v2 that comes with less layers so we can add new ones
the dev versions with 6.5 already have node 18 and php 8.2

regarding the zip file...yes...this is meant if you want to install a shopware version by selecting one from the shopware website. this is only a small helper that downloads a zip and extracts it to the docroot. and it would still work for older shopware versions

6.5 can also be installed, by using the new way of shopware using the composer create-project.
in the end, essentials is a prepared lamp environment to run dockware..."make download" is only a nice addon

i hope this helps for now :)

**Comment by grosssmutttr** [comment]
Hi there,

thanks a lot for this! Haven't tried the composer create-project but it is working now as I found out that I have to rename .htaccess.dist to .htaccess in the public folder for Shopware 6.4.20.2

Now I want to setup a shopware local project where I can install my own shopware 6.5 shop (so not the dev image -> essentials) and use the build-administration/build-storefront. 
Is it correct that this is not possible due to the missing NODE_JS version 18?

When I define it the container logs the following:
2023-05-26 18:30:32 shopware6-5 | ! WARNING: Version '18' does not exist.
2023-05-26 18:30:32 shopware6-5 | default -> 18 (-> N/A)

So 18 is not available?
If I'm correct and you update the image to be able to use NODE_JS = 18 can I make an update of the image:
docker pull dockware/essentials:latest

Are the docker containers reset with this method and do I need to setup the project again or does this not affect the files/my plugins I synced with docker?

**Comment by grosssmutttr** [comment]
Now I tried to install NODE_JS 18 in the essentials container.
As Ubuntu 18 is used there is no support for NODE_JS 18 so I tried to upgrade Ubuntu - but unfortunately no chance as there is no systemd - so we have to wait for a new image with NODE_JS 18 support...

**Comment by martinweinmayr** [comment]
Essentials:2.0 also have node 18 and php 8.2 yyy

**Comment by grosssmutttr** [comment]
@martinweinmayr 
thx a thousand times :-)
That's it - perhaps the documentation should be updated and instead of essentials:latest it should mention essential:2.0 - I really thought "latest" also means latest :-)

**Comment by martinweinmayr** [comment]
We will tag 2.0 as latest when it is out there for 3-4 weeks as it is a total rewrite.

Sorry for the confusion

**Comment by grosssmutttr** [comment]
Ok now it gets really confusing. I setup a shopware 6.4 project with this:
```
version: "3"

services:
  shopware-64:
    image: dockware/essentials:2.0
    container_name: shopware-64
    ports:
      - "80:80"
      - "3306:3306"
      - "22:22"
      - "8888:8888"
      - "9999:9999"
      - "9998:9998"
    networks:
      - web
    environment:
      - XDEBUG_ENABLED=1
      - PHP_VERSION=8.2
      - NODE_VERSION=16 #12|14|16=shopware6.4|18=shopware6.5

networks:
  web:
    external: false
```
After working with my project and shutting down my pc I wanted to continue my work.
Now I cannot start the container as it states ! WARNING: Version '16' does not exist. 
So the NODE_JS Version 16 is not available and now I have to start again.

If I would start the project with the essentials:latest this would be deprecated in 2-3 weeks.
Which NODE_JS Versions are in the essentials:2.0?

**Comment by martinweinmayr** [comment]
https://github.com/dockware/dockware/blob/features/v2/CHANGELOG.md

We make a cut and will only maintain new stuff. 
So 2.0 only supports 18 and newer in the future. 

Is this a problem? do you need 16?

**Comment by grosssmutttr** [comment]
yeah because we also have to develop the shopware 6.4 plugins for some times because our clients want to wait before migrating to shopware 6.5 :-)

**Comment by martinweinmayr** [comment]
ok so if we bring the old essentials 1.0 with php 8.2 on board it would be also fine, right?

**Comment by grosssmutttr** [comment]
Just tried to use the watch-admin make command in the essentials:2.0 image - not available...
Only these commands:
status                         status infos
xdebug-on                      enables xdebug
xdebug-off                     disables xdebug
permission-repair              repairs the general file permissions
switch-php                     Switches to another supported PHP version, [make switch-php version=7.4]

**Comment by grosssmutttr** [comment]
@martinweinmayr 
Am I doing something wrong or is the watch-administration not implemented in essentials 2.0?
If so - is it implemented in the future?
